### PR TITLE
admin-guide: update examples etc for flux-core 0.50.0 release

### PR DESCRIPTION
**Pull request by garlick**
We haven't updated the admin guide in a while, so I made a pass through it.

### CODE REVIEW

File: guides/admin-guide.rst [hunk 1]
```diff
@@ -13,10 +13,10 @@ resource manager on a cluster.
     in this guide may change with regularity.
 
     This document is in DRAFT form and currently applies to flux-core
-    version 0.45.0.
+    version 0.50.0.
```
Comment (vsoch) [inline guides/admin-guide.rst:16]: Whoa! Are we really nearing 50?! I hope we can get that bugfix in for zeromq - I would be a very happy dinosaur.

File: guides/admin-guide.rst [hunk 2]
```diff
@@ -1082,23 +1095,20 @@ EXPEDITE or HOLD.
...
 
 .. code-block:: console
 
- $ flux job cancelall
- flux-job: Command matched 5 jobs (-f to confirm)
- $ flux job cancelall -f
- flux-job: Canceled 5 jobs (0 errors)
+ $ flux cancel --user=all --dry-run
+ flux-cancel: Would cancel 3 jobs
+ $ flux cancel --user=all
```
Comment (vsoch) [inline guides/admin-guide.rst:1108]: Just to double check - this doesn't require the `-f` for force?

File: guides/admin-guide.rst (hunk 1 above)
Comment (garlick) [reply guides/admin-guide.rst:16]: I've proposed it and tentatively added it to the release notes for 50.

File: guides/admin-guide.rst (hunk 2 above)
Comment (garlick) [reply guides/admin-guide.rst:1108]: Right.  Now we have `--dry-run` to show what the command is going to do without doing it.  The default is to do stuff now.

Comment (vsoch) [reply guides/admin-guide.rst:1108]: oh nice! That is a a _hugely_ better design. +1 !

**Review by grondo — APPROVED** [review]
Nice update, thanks!

**Comment by garlick** [comment]
Thanks!  Setting MWP.

**Pull request merged**